orm/models.py

```diff
@@ -110,13 +110,13 @@ def filter(self, **kwargs):
 
                 model_cls = self.model_cls
                 if related_parts:
-                    # Add any implied select_related
+                    #  Add any implied select_related
                     related_str = "__".join(related_parts)
                     if related_str not in select_related:
                         select_related.append(related_str)
 
                     # Walk the relationships to the actual model class
-                    # against which the comparison is being made.
+                    #  against which the comparison is being made.
```

Comment:
These whitespaces look unnecessary.

Reply:
Yes! Black reformatted the code in all 3 PRs, and for some reason it added an extra space there which I forgot to remove here.